engine/graphics/src/wscript

```diff
@@ -48,9 +48,9 @@ def build(bld):
         obj.source.remove(os.path.join('vulkan','graphics_native.cpp'))
         obj.source.append(_get_native_source(build_util.get_target_os(), 'vulkan'))
 
-        if bld.env.PLATFORM == 'x86_64-darwin':
+        if bld.env.PLATFORM in ('x86_64-macos','x86_64-macos'):
             obj.source.append('vulkan/macosx/graphics_vulkan_macosx_surface.mm')
-        elif bld.env.PLATFORM == 'armv7-darwin' or bld.env.PLATFORM == 'arm64-darwin':
+        elif bld.env.PLATFORM in ('arm64-ios',):
```

Comment:
I was at first confused by why you left a trailing comma, until I realised that you used a tuple and not a list. I had to look that up to learn that tuples with 0 or 1 elements require this extra comma to function as intended.

I had not thought of this before and would probably have used a list myself as that is what I was familiar with :-)

Reply:
True. My guess would be that they are more performant because they are immutable.